# 为什么愤怒函数超过了调用栈大小？

> 原文：<https://dev.to/michaeljolley/why-did-the-angry-function-exceed-the-callstack-size-1fdk>

它和自己发生了争执